pkg/controllers/register.go

```diff
@@ -165,6 +172,21 @@ func RegisterControllers(ctx *context.ControllerContext, syncers []syncer.Object
 	return nil
 }
 
+func registerInitManifestsController(ctx *context.ControllerContext) error {
+	controller := &manifests.InitManifestsConfigMapReconciler{
+		Client:   ctx.LocalManager.GetClient(),
+		Log:      loghelper.New("initmanifests-controller"),
+		VManager: ctx.VirtualManager,
```

Comment:
Please rename VManager to `VirtualManager` as we are using it like this everywhere else